tests/base.py

```diff
@@ -60,6 +60,7 @@ def get_properties(self, original: bool = True):
                 }
             ]
         }
+        self.account_id = os.getenv("TAP_TWITTER_ADS_ACCOUNT_IDS").split(" ")[0]
```

Comment:
If we expect the comma-separated list of account ids in the config the why are we splitting account_ids by `" "`?

Reply:
Updated base.py to split account_ids by `" "`.